canonical/charmed-kubeflow.io
# Update navigation with new branding

## Done

Update navigation with new branding and the title "Canonical Kubeflow"

## QA

- Visit https://charmed-kubeflow-io-121.demos.haus/
- See that the new branding has been applied
- View on mobile and desktop, verify the nav works

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5644
